docs/source/data_ingestion_and_preparation/using_jupyter.md

```diff
@@ -1,29 +1,25 @@
 # Using Jupyter to create data preparation jobs
 
-It can be useful to use a [Jupyter notebook](https://jupyter.org/) to write and iterate on custom data preparation code. By default, Sycamore includes a Jupyter notebook configured with Sycamore’s data preparation libraries and other dependencies, and you can easily use this to create scripts in a notebook or run data preparation code from the Jupyter shell. You can also install and configure Jupyter with the Sycamore libraries locally.
+Using a [Jupyter notebook](https://jupyter.org/) makes it easy to write and iterate on custom data preparation code. By default, Sycamore includes a containerized Jupyter notebook configured with Sycamore’s data preparation libraries and other dependencies. You can easily use this to create and run scripts in a notebook or run data preparation code from the Jupyter shell. You can also [install and configure Jupyter locally](###running-jupyter-locally).
 
 ## Using the Jupyter container
 
 ### Jupyter notebook
 
-Sycamore launches a default container with Jupyter. In order to maintain security, Jupyter uses a token to limit access to the notebook. For the first 5 minutes the Jupyter container will periodically print out instructions for connecting. If you can't see them, you can get them by running:
+Sycamore includes a default container with Jupyter. In order to maintain security, Jupyter uses a token to limit access to the notebook. For the first 5 minutes after launching, the Jupyter container will periodically print out instructions for connecting. If you can't find these instructions, you can get them by running:
 
 `docker compose logs jupyter`
```

Comment:
docker compose logs jupyter | grep Visit

works slightly better since it finds the instructions even if Jupyter has logged lots of crap out.

Reply:
Ack